python-mini-projects/projects/Get_meta_information_of_images/gps_utils.py

```diff
@@ -0,0 +1,23 @@
+import exifread
+import requests
+from geopy.geocoders import Nominatim
+
+def format_lati_long(data):
+	list_tmp=str(data).replace('[', '').replace(']', '').split(',')
+	list=[ele.strip() for ele in list_tmp]
+	if (list[-1].find('/') != -1):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you do not need the position returned by the [`find()`](https://docs.python.org/3/library/stdtypes.html#str.find) function. Consider using the operators [`in` and `not in`](https://docs.python.org/3/reference/expressions.html#membership-test-details) to check for simple membership, or more specific operations [`startswith()`](https://docs.python.org/3/library/stdtypes.html#str.startswith) and [`endswith()`](https://docs.python.org/3/library/stdtypes.html#str.endswith) to check for prefix and suffix relationships with the string.